app/components/treatment/Treatment.js

```diff
@@ -33,58 +38,49 @@ const Treatment = ({ plan, treatment }) => {
   }
 
   return (
-    <div className='container'>
-      <Helmet title={treatment.exercise.title} />
-      <div className='row'>
-        <div className='col-md-12'>
-            <div className='row'>
-              <h1> { `${treatment.exercise.title}` } </h1>
-            </div>
-        </div>
-        <div className='col-md-9'>
-          <div className='row'>
-            <Paper style={ padded }>
-              <div className='row'>
-                <div className='col-md-12'>
-                  { media }
-                </div>
-              </div>
-              <div className='row'>
-                <div className='col-md-6'>
-                  <div>
-                    <p style={ emphasis }>Description: </p>
-                    <p>{ treatment.exercise.description }</p>
-                  </div>
-                </div>
-                <div className='col-md-6'>
-                  <div>
-                    <p><span style={ emphasis }>Total Time (All Sets):  </span>
-                    { treatment.time_per_exercise }</p>
-                  </div>
-                  <div>
-                    <p><span style={ emphasis }>Sets:  </span>
-                    { treatment.sets }</p>
-                  </div>
-                  <div>
-                    <p><span style={ emphasis }>Reps:  </span>
-                    { treatment.reps }</p>
-                  </div>
-                  <div>
-                    <p><span style={ emphasis }>Resistance:  </span>
-                    { treatment.resistance }</p>
-                  </div>
-                  <div>
-                    <p style={ emphasis }>Therapist Notes: </p>
-                    <p>{ treatment.notes }</p>
-                  </div>
-                </div>
-              </div>
-            </Paper>
-          </div>
-        </div>
-        <div className='col-md-3'>
-          <PlanPanel plan={ plan } />
-        </div>
+    <div id='single-treatment'>
+      <Helmet title={ treatment.exercise.title } />
+      <h1 className='page-header'> { treatment.exercise.title } </h1>
+      <div className='treatment-content'>
+        <SidePanel>
+          <span><h4> { treatment.exercise.title } </h4></span>
+          <InfoItem icon="assignment" label="Description" iconColor={ primary }
+            content={ treatment.exercise.description } />
+          <InfoItem icon="all_inclusive" label="Sets" iconColor={ primary }
+            content={ treatment.sets } />
+          <InfoItem icon="layers" label="Reps" iconColor={ primary }
+            content={ treatment.reps } />
+          <InfoItem icon="alarm" label="Time" iconColor={ primary }
+            content={ formatTime(treatment.time_per_exercise) } />
+          <InfoItem icon="fitness_center" label="Resistance" iconColor={ primary }
+            content={ treatment.resistance } />
+          {
+            treatment.notes ?
+              <InfoItem icon="speaker_notes" label="Notes"
```

Comment:
This is the only black icon in this section. Not sure if this was intended, just wanted to make sure you were aware